pkg/kubeconfig/kubeconfig.go

```diff
@@ -43,25 +43,28 @@ type configLoader struct {
 
 func newConfigLoader(existing *cluster.Cluster) (*configLoader, error) {
 	pubKeyPath := local.Expand(existing.SSH.PublicKeyPath)
-	// --------------------------------------------------------------------------------
-	//
-	// @kris-nova
-	//
-	// We don't need to check a key first, because SSH can support multiple auth implementations
-	// So here we just add BOTH a key based auth, and an arbitrary agent. Please don't touch this
-	// without talking to @kris-nova first OR unless something is just completely fucked
```

Comment:
We probably shouldn't get rid of this comment -- I remember @kris-nova went through some grief to add the second agent.